pkg/handler/snapshots.go

```diff
@@ -327,6 +330,51 @@ func (sh *SnapshotHandler) bulkDeleteSnapshot(c echo.Context) error {
 	return c.NoContent(http.StatusNoContent)
 }
 
+// GetTemplateRepoConfigurationFiles godoc
+// @Summary      Get configuration files for all repositories in a template
+// @ID           getTemplateRepoConfigurationFiles
+// @Tags         repositories
+// @Accept       json
+// @Produce      text/plain
+// @Param        template_uuid  path  string    true  "Identifier of the template"
+// @Success      200   {string} string
+// @Failure      400 {object} ce.ErrorResponse
+// @Failure      401 {object} ce.ErrorResponse
+// @Failure      404 {object} ce.ErrorResponse
+// @Failure      500 {object} ce.ErrorResponse
+// @Router       /templates/{template_uuid}/config.repo [get]
+func (sh *SnapshotHandler) getTemplateRepoConfigurationFiles(c echo.Context) error {
```

Comment:
What are you thoughts on moving this into the template handler? I feel like it might be better suited there since the url path is unrelated to snapshots

Reply:
yea i think so too, i put it here to be with the other config.repo endpoints but was questioning that 😄 will move it!